src/training/main.py

```diff
@@ -277,6 +287,8 @@ def main(args):
         if args.ddp_static_graph:
             # this doesn't exist in older PyTorch, arg only added if enabled
             ddp_args['static_graph'] = True
+        if args.use_bnb_linear is not None:
```

Comment:
couldn't this be put right after replace_linear() in the previous `if use_bnb_linear` block?

Reply:
@mitchellnw one q here

Reply:
yep good point that simplifies -- i'll make the change and run a quick test to make sure all good.

Reply:
fixed, thanks @rwightman!